src/Tizen.Log/Tizen/Log.cs

```diff
@@ -112,17 +114,52 @@ public static void Fatal(string tag, string message, [CallerFilePath] string fil
             Print(Interop.Dlog.LogPriority.DLOG_FATAL, tag, message, file, func, line);
         }
 
-        static void Print(Interop.Dlog.LogPriority priority, string tag, string message, string file, string func, int line)
+        static unsafe void Print(Interop.Dlog.LogPriority priority, string tag, string message, string file, string func, int line)
         {
-            if (String.IsNullOrEmpty(file))
-            {
-                Interop.Dlog.Print(priority, tag, "%s", message);
-            }
-            else
+            int byteCountTag = Encoding.UTF8.GetMaxByteCount(tag.Length);
+            Span<byte> tagBuffer = byteCountTag < 1024 ? stackalloc byte[byteCountTag + 1] : new byte[byteCountTag + 1];
```

Comment:
for the readability, I recommand to rename variable names

```suggestion
            int tagByteLength = Encoding.UTF8.GetMaxByteCount(tag.Length);
            Span<byte> tagByte = byteCountTag < 1024 ? stackalloc byte[tagByteCount + 1] : new byte[tagByteCount + 1];
```

Reply:
Thank you. I fixed those variable name.